srtcore/list.cpp

```diff
@@ -97,15 +97,7 @@ srt::CSndLossList::~CSndLossList()
 
 void srt::CSndLossList::traceState() const
 {
-    int pos = m_iHead;
-    while (pos != SRT_SEQNO_NONE)
-    {
-        std::cout << pos << ":[" << m_caSeq[pos].seqstart;
-        if (m_caSeq[pos].seqend != SRT_SEQNO_NONE)
-            std::cout << ", " << m_caSeq[pos].seqend;
-        std::cout << "], ";
-        pos = m_caSeq[pos].inext;
-    }
+    traceState(std::cout);
```

Comment:
```suggestion
    traceState(std::cout) << '\n';
```
And remove `std::cout << "\n";` below.

Reply:
What does this change exactly?

Reply:
Localizes the logic in a single line.